packages/accessibility/src/accessibleTarget.js

```diff
@@ -62,4 +62,31 @@ export const accessibleTarget = {
      * @todo Needs docs.
      */
     _accessibleDiv: false,
+
+    /**
+     * Specify the type of div the accessible layer is. Screen readers treat the element differently
+     * depending on this type. Defaults to button.
+     *
+     * @member {string}
+     * @memberof PIXI.DisplayObject#
+     */
+    accessibleType: 'button',
+
+    /**
+     * Specify the pointer-events the accessible div will use
+     * Defaults to auto.
+     *
+     * @member {string}
+     * @memberof PIXI.DisplayObject#
+     */
+    accessiblePointerEvents: 'auto',
```

Comment:
fair point! will tweak and test, ya think we would we end up with double events if the click goes through the div and onto the canvas?

Reply:
Right now with auto, the click gets to the div, and then stops there; the canvas doesn't receive it. 
With none, the click bypasses the div and goes onto the canvas.

So shouldn't get double events

Reply:
That makes sense, one more question :)
Would change not adversely affect mobile accessibility you think? As you need to tap to select?

Reply:
You may be right! I have a certain level of uncertainty round this too. With accessibility on mobile the divs need to be sort of  'double tapped'. Not sure if it would throw a spanner in the works!

I think this is cool to merge for now, but will test to see if 'none' is a better approach!